### PR TITLE
chore: use context for env variables

### PR DESCRIPTION
**Pull request by RitamAgrawal**
### What does this PR do?
Use context to define all environment variables

### What issues does this PR fix or reference?
@W-10539922@

### CODE REVIEW

**Comment by mohanraj-r** [comment]
@RitamAgrawal Can you please try if this runs with [CircleCI Local CLI](https://circleci.com/docs/2.0/local-cli/)

File: .circleci/config.yml [hunk 1]
```diff
@@ -273,15 +273,17 @@ workflows:
...
             branches:
               only: main
           requires:
             - prepublish
             - hold
       - publish-plugin:
-          context: pdt-publish-restricted-context
+          context:
+            - IEAT_keys
+            - AWS
```
Comment (gbockus-sf) [inline .circleci/config.yml:286]: So why is this one different?

**Review by mohanraj-r — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (RitamAgrawal) [reply .circleci/config.yml:286]: @gbockus-sf the only job that requires AWS credentials is publish-plugin. AWS credentials are defined in a separate context named AWS, which is shared by teams across forcedotcom.

**Review by gbockus-sf — COMMENTED** [review]
👍

**Pull request merged**